app/controllers/webhooks/github_controller.rb

```diff
@@ -39,9 +39,12 @@ def push_event(chat, body)
     end
 
     msg = msg % [body['pusher']['name'], body['commits'].length, body['compare']]
+    body['commits'].each do |commit|
+      msg += "\n%s - %s" % [commit['id'][0..5], commit['message']]
```

Comment:
[0..5] means the first 6 characters of the commit id. The .each operates on all the commits that get sent in the push_event, which I _believe_ to be all the commits that haven't been pushed yet. I'm not limiting how many commits are shown here.

So it looks like:
123fba - Done this
aab21a - Done that